src/redux/selectors/index.js

```diff
@@ -384,3 +388,88 @@ export const timelineIntervalsSelector = createSelector(
     };
   },
 );
+
+export const accordionsContainerDataSelector = createSelector(
+  [
+    filteredResourceTypesSelector,
+    selectedResourceTypeSelector,
+    showCollectionOnlySelector,
+    showMarkedOnlySelector,
+    (_, ownProps) => ownProps,
+  ],
+  (
+    filteredResourceTypes,
+    selectedResourceType,
+    showCollectionOnly,
+    showMarkedOnly,
+    ownProps,
+  ) => {
+    const { fromContentPanel, fromCatalogScreen } = ownProps;
```

Comment:
is it valid to have a `fromContentPanel` and  `fromCatalogScreen` value at the same time?   (or: if one is false, the other must be true?)

if not, I wonder if there should be a single variable.

Reply:
makes sense. I can refactor.